src/pages/docs/tutorials/tutorial-hello-enact/adding-css/index.md

```diff
@@ -3,13 +3,11 @@ title: Adding CSS
 github: https://github.com/enactjs/docs/blob/develop/src/pages/docs/tutorials/tutorial-hello-enact/adding-css/index.md
 order: 2
 ---
-With our [basic Hello Enact!](../basics/) app built and running, we can start to expand
-it by adding some styling. The first stop is defining and managing [CSS classes in React](#css-classes-in-react) followed by exploring [CSS modules](#introducing-css-modules).
+With our [basic Hello Enact!](../basics/) app built and running, we can start to expand it by adding some styling. The first stop is defining and managing [CSS classes in React](#css-classes-in-react) followed by exploring [CSS modules](#introducing-css-modules).
```

Comment:
I think it's okay to have a changing line here. The raw file looks better with it.

Reply:
OK. 
I agree to keep the line breaks.